projects/core/src/lib/lesson/lesson.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="lesson" *ngIf="lesson" [class.lesson__selected]="selected" (click)="select()">
+	<div class="lesson__description">
+		<h4 class="lesson__title">
+			<i class="lesson__chevron-down fa-solid fa-chevron-down" *ngIf="!selected"></i>
+			<i class="lesson__chevron-up fa-solid fa-chevron-up" *ngIf="selected"></i>
+			Module #{{ lesson.id }}: {{ lesson.title }}
+		</h4>
+		<div class="lesson__details" *ngIf="selected">
+			<hr class="lesson__separator" />
+			<div class="lesson__resources resources">
+				<h4 class="lesson__title">Self-study Materials</h4>
+				<p class="resources__description">
+					Please, get prepared in accordance with the materials provided in the attached
+					file.
+				</p>
+				<p class="resources__resource resource" *ngFor="let resource of lesson.resources">
+					<i class="resource__paperclip fa-solid fa-paperclip" *ngIf="resource"></i>
+					<a class="resource__link" href="{{ resource.url }}" target="_blank">
+						{{ resource.name }}
+					</a>
+				</p>
+			</div>
+			<hr class="lesson__separator" />
+			<div class="lesson__meeting meeting">
```

Comment:
.meeting class is missing from component's style rules

Reply:
ok, i removed this class